exercises/beer-song/BeerSongExample.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  beer-song.swift
+//  xSwift
+//
+//  Created by Alwyn Savio Concessao on 01/12/16.
+//  Copyright © 2016 exercism.io. All rights reserved.
+//
+class BeerSong{
+
+    let numberOfBeerBottles = 99
+    
+    
+    func generateVersesOfBeerSong(){
+    
+        var numberOfBottlesOfBeer = numberOfBeerBottles
+        while(numberOfBottlesOfBeer > 2){
+    
+    print("\(numberOfBottlesOfBeer) bottles of beer on the wall,\(numberOfBottlesOfBeer) bottles of beer.")
+    numberOfBottlesOfBeer -= 1
+    print("Take one down and pass it around,\(numberOfBottlesOfBeer) bottles of beer on the wall.")
```

Comment:
Add a space after this comma (should be `pass it around, \(numberOfBottlesOfBeer)`).